packages/gestalt/src/Avatar.js

```diff
@@ -69,95 +67,82 @@ const sizes = {
   lg: 72,
 };
 
-export default class Avatar extends React.PureComponent<AvatarProps, State> {
-  static propTypes = {
-    name: PropTypes.string.isRequired,
-    outline: PropTypes.bool,
-    src: PropTypes.string,
-    size: PropTypes.oneOf(['sm', 'md', 'lg']),
-    verified: PropTypes.bool,
-  };
-
-  state = {
-    isImageLoaded: true,
-  };
+export default function Avatar(props: Props) {
+  const [isImageLoaded, setIsImageLoaded] = React.useState(true);
```

Comment:
This is a bad var name. We should follow up with another diff that renames to `imageHasFailedToLoad = false`.